float64[] equilibrium_pose
---
bool success